style.css

```diff
@@ -0,0 +1,425 @@
+  margin-bottom: 36px;
+  left: 789px;
+  font-family: Montserrat, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 80px;
+  line-height: 82px;
+  color: #000;
+}
+
+.new-collection__description {
+  width: 508px;
+  margin-bottom: 36px;
+  font-family: Montserrat, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 17px;
+  line-height: 25px;
+  color: #000;
+}
+
+.new-collection__farther {
+  width: 203px;
+  height: 44px;
+  margin-right: 54px;
+  font-family: Montserrat, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 0;
+  background: none;
+  text-transform: uppercase;
+  color: #000;
+  cursor: pointer;
+  padding: 14px 41px;
+  border: 1.5px solid #000;
+  box-sizing: border-box;
+}
+
+.new-collection__farther:hover {
+  background: #d3d3d3;
+}
+
+.new-collection__other {
+  position: relative;
+  display: inline-block;
+  outline: 0;
+  border: 0;
+  font-family: Montserrat, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  background: none;
+  text-transform: uppercase;
+  color: #000;
+  cursor: pointer;
+}
+
+.new-collection__other::after {
+  content: "";
+  position: absolute;
+  background-image: url(./img/Vector%202.1.png);
+  top: 4px;
+  left: 225px;
+  width: 44px;
+  height: 12px;
+}
+
+.new-collection__other:hover {
+  transform: scale(1.2);
+}
+
+.figure {
+  position: absolute;
+  left: 50%;
+  bottom: 81px;
+}
+
+.figure__scrooll {
+  position: absolute;
+  width: 27px;
+  height: 46px;
+}
+
+.figure__strip {
+  position: absolute;
+  width: 3px;
+  height: 8.21px;
+  margin: 9px 10px 8px 11px;
+}
+
+.social-networks {
+  position: absolute;
+  bottom: 41px;
+  left: 42px;
+  transition: 1s;
+  cursor: pointer;
+}
+
+.social-networks:hover {
+  transform: scale(1.2);
+}
+
+.social__link,
+.social__link-vk {
+  margin-right: 13px;
+}
+
+@media (max-width: 565px) {
+  .adaptive {
+    display: none;
+}
+
+.header__language {
+  display: none;
+}
+
+.header::before {
+  display: none;
+}
+
+  .new-collection__girl {
+  display: none;
+}
+
+  .social-networks {
+  display: none;
+}
+
+  .figure {
+  display: none;
+}
+
+.header__title {
+  width: 98px;
+  height: 37px;
+  left: 26px;
+  margin: 14px 250px 18px 26px;
+  font-weight: 500;
+  font-size: 30px;
+  line-height: 37px;
+  text-align: center;
+  letter-spacing: 0.07em;
+    color: #000;
+}
+
+  .header__menu {
+  display: block;
+ }
+
+  .new-collection {
+    background-image: url(./img/girl_mobile.png);
+    background-repeat: no-repeat;
+    width: 565px;
+    height: 583px;
+}
+
+  .new-collection__title {
+  position: relative;
+    width: 297px;
+  height: 103px;
+  left: 26px;
+  top: 289px;
+  font-weight: bold;
+  font-size: 50px;
+  line-height: 51px;
+}
+
+.new-collection__title::after {
+  content: "";
+  position: absolute;
+  background-image: url(./img/Vector1.png);
+  width: 35px;
+  height: 20px;
+  left: 145px;
+  top: 115px;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Locc/pullRequest?prid=5986341)